src/config/types.ts

```diff
@@ -322,6 +339,13 @@ export type AvailableDates = {
     | PointDataLayerProps['id']]: number[];
 };
 
+/* eslint-disable camelcase */
```

Comment:
It's best to either:
- only deactivate it for the one line where it's needed
- reactivate it after it's not needed
- deactivate it at the top of the file if needed throughout the file

Reply:
Done. Included in e52480930a512a73b0f8fc8280e220e8f7e88ae0